app/actions/remote/channel.ts

```diff
@@ -1107,6 +1108,12 @@ export async function switchToChannelById(serverUrl: string, channelId: string,
     markChannelAsRead(serverUrl, channelId);
     fetchChannelStats(serverUrl, channelId);
 
+    getChannelById(database, channelId).then((channel) => {
```

Comment:
@avinashlng1080 / @enahum - I've moved the `fetchGroupsForChannel` to the remote channel switch action, but I only need to call it if the channel is group constrained - it made more sense having it in the local action as the `ChannelModel` was already available there but do check this and let me know if it works or if there's a better way -